drivers/wifi/nrf700x/zephyr/src/zephyr_work.c

```diff
@@ -77,6 +88,25 @@ static int workqueue_init(void)
 						K_THREAD_STACK_SIZEOF(irq_wq_stack_area),
 						CONFIG_NRF700X_IRQ_WQ_PRIORITY,
 						NULL);
+#ifdef CONFIG_NRF700X_TX_DONE_WQ_ENABLED
+	k_work_queue_init(&zep_wifi_tx_done_q);
+
+	k_work_queue_start(&zep_wifi_tx_done_q,
+						tx_done_wq_stack_area,
```

Comment:
Indentation

Reply:
Tried again, aligning to open brace vs. not exceeding 100 lines vs. using tabs as much as possible, its impossible to do this alignment anyother (AFAIK), so, closing this comment as checkpatch is passing.